app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -95,6 +106,20 @@ class StateFragmentPresenter @Inject constructor(
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
       logger.d("StateFragment", "getCurrentState: ${result.state.name}")
+      entity_type = "exploration"
+      // TODO replace exploration Id from result. Exploration id is missing in proto
+      entity_id = "DIWZiVgs0km-"
+      val customizationArgsMap: Map<String, InteractionObject> = result.state.interaction.customizationArgsMap
+      logger.d("TAG", "getCurrentState: " + result.state)
```

Comment:
Please create a variable TAG initialised as "StateFragment" and use throughout the fragment

Reply:
Done.